src/convertToObject.js

```diff
@@ -12,7 +14,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+
+  const stylesStringSplitted = stylesString.split(';');
+
+  const trimmedString = stylesStringSplitted
+    .map(string => string.split(':')
```

Comment:
```suggestion
  const stylesStringSplitted = stylesString.split(';')
    .map(string => string.split(':')
```
Можна зразу ж ланцюжком прописати всі методи